draft-ietf-quic-recovery.md

```diff
@@ -615,7 +615,7 @@ a packet containing unacknowledged CRYPTO data earlier than the PTO expiry,
 subject to address validation limits; see Section 8.1 of {{QUIC-TRANSPORT}}.
 
 Peers can also use coalesced packets to ensure that each datagram elicits at
-least one acknowledgement.  For example, clients can coalesce an Initial packet
+least one acknowledgement.  For example, a client can coalesce an Initial packet
```

Comment:
The previous sentence uses a plural. Why change one but not the other?

Reply:
This is followed by "a server". Why are both singular and plural used in the same sentence?

Reply:
I think that "a client" and "a server" are best here.  The previous sentence is general, this is an example and therefore specific.

That said, we are almost certainly hugely inconsistent about addressing requirements to "endpoints" (plural) or "an endpoint" (singular).